src/app/editors/editors.component.html

```diff
@@ -11,5 +14,7 @@
     <app-editor [file]="file" *ngIf="!file.collapsed" (onCodeChange)="onCodeChange({file: file, code: $event})">
     </app-editor>
   </div>
+
+  <resize [(width)]="editorWidth"></resize>
```

Comment:
Do you think we could use content projection here?
This way we can access `app-editors` component using @ContentChild and measure/resize it from the resize component without modifying app-editors code at all.

https://scotch.io/tutorials/angular-2-transclusion-using-ng-content